python/lint/flake8.ini

```
@@ -2,4 +2,4 @@
 max-line-length = 100
 max-complexity = 10
 exclude = *build*,*artifacts*
-ignore = D100,D101,D102,D103,D104,D105,D106,D107,D203,D213,W504
+ignore = D100,D101,D102,D103,D104,D105,D106,D107,D203,D213,W504,E203,W503
```

ha, interesting: https://github.com/psf/black/issues/315#issuecomment-395457972 and [flake8rules.com/rules/W503.html](https://www.flake8rules.com/rules/W503.html#:~:text=This%20rule%20goes%20against%20the%20PEP%208%20recommended%20style)

could you sort it though?
```suggestion
ignore = D100,D101,D102,D103,D104,D105,D106,D107,D203,D213,E203,W503,W504
```